File: URI/Ruby/uri-1012.rb
#URI - 1012: https://www.urionlinejudge.com.br/judge/pt/problems/view/1012

Input = gets.split(' ')
A = Input[0].to_f
B = Input[1].to_f
C = Input[2].to_f

Triangle_area = A * C / 2
Circle_area = 3.14159 * C**2
Trapezoid_area = (A + B) * C / 2
Square_area = B**2
Rectangle_area = A * B

puts "TRIANGULO: #{'%.3f' % Triangle_area}"
puts "CIRCULO: #{'%.3f' % Circle_area}"
puts "TRAPEZIO: #{'%.3f' % Trapezoid_area}"
puts "QUADRADO: #{'%.3f' % Square_area}"
puts "RETANGULO: #{'%.3f' % Rectangle_area}"

File: URI/C#/uri-1012.cs
// URI - 1012

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] values = Console.ReadLine().Split(' ');

            ShowAreas(values);
        }

        private static void ShowAreas(string[] values)
        {
            double aValue = double.Parse(values[0]);
            double bValue = double.Parse(values[1]);
            double cValue = double.Parse(values[2]);

            Console.WriteLine("TRIANGULO: " + ((aValue * cValue) / 2).ToString("F3"));
            Console.WriteLine("CIRCULO: " + (3.14159 * Math.Pow(cValue, 2)).ToString("F3"));
            Console.WriteLine("TRAPEZIO: " +((aValue + bValue) * cValue / 2).ToString("F3"));
            Console.WriteLine("QUADRADO: " + (Math.Pow(bValue, 2)).ToString("F3"));
            Console.WriteLine("RETANGULO: " + (aValue * bValue).ToString("F3"));
        }
    }
}